cmd/earthly/flags.go

```diff
@@ -188,6 +188,13 @@ func (app *earthlyApp) rootFlags() []cli.Flag {
 			Destination: &app.logstreamDebugManifestFile,
 			Hidden:      true, // Internal.
 		},
+		&cli.StringFlag{
+			Name:        "logstream-address",
```

Comment:
I've been using this flag to setup an integration test between the CLI and cloud.
It's been instrumental in diagnosing and fixing issues.